web/template/partial/stream/chat/chat.gohtml

```diff
@@ -32,42 +33,27 @@
          @reorder="c.sortMessages()"
          @keyup.escape="c.current.replyTo=0"
          id="chatWrapper"
-         class="flex relative flex-col text-1 bg-white dark:bg-secondary h-full border-l z-40 dark:border-gray-800">
+         class="flex relative flex-col text-1 h-full">
         <!-- Chat header -->
```

Comment:
<img width="285" alt="grafik" src="https://user-images.githubusercontent.com/30267166/192623755-168f3165-dfaa-49b6-bdf5-7325283a1c0b.png">

I really like the separators in the light theme, it really brings the chat header together. Maybe we can replicate that in dark mode?

Reply:
Done.

<img width="439" alt="image" src="https://user-images.githubusercontent.com/29633518/193506711-ec8e344d-1e43-414a-ad6d-1a2d090255a7.png">

Reply:
Looks really nice!